tips/033_semantic_validators.md

```diff
@@ -0,0 +1,262 @@
+## Effective AI Engineering #33: Semantic Validators
+
+**Are your AI responses technically correct but completely missing the point?** Your model generates grammatically perfect text that doesn't answer the user's actual question or contradicts known facts.
+
+Traditional validation catches syntax errors and format issues, but semantic correctness requires understanding meaning, context, and factual accuracy. Human reviewers can spot these issues instantly, but manual validation doesn't scale.
+
+### The Problem
+
+Many developers only validate AI outputs for format and basic safety without checking semantic accuracy. This creates challenges that aren't immediately obvious:
```

Comment:
I think we need to call out that sometimes we want validation that is hard to specify because it is fuzzier than what you might apply more traditional validation to.